### ISSUE TITLE
Identical methods for DEB installation

### ISSUE THREAD

**Issue by viktor-klestov** [ClickHouse/ClickHouse#35977]
It looks like there is a problem in installation chapter of docs. Both methods are identical and likely deprecated as apt-key is not recommended anymore.
![image](https://user-images.githubusercontent.com/82416965/161861858-d4646eb6-26ac-4c2a-b69f-3d57d66958c1.png)

**Comment by rfraposa** [comment]
The first way has been updated in the docs to the most recent install script [here](https://github.com/ClickHouse/ClickHouse/blob/master/docs/_includes/install/deb.sh), but it is not clear to me if this resolves the issue. For example, it still uses `apt-key`: 

```
sudo apt-get install -y apt-transport-https ca-certificates dirmngr
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754

echo "deb https://packages.clickhouse.com/deb stable main" | sudo tee \
    /etc/apt/sources.list.d/clickhouse.list
sudo apt-get update

sudo apt-get install -y clickhouse-server clickhouse-client

sudo service clickhouse-server start
clickhouse-client # or "clickhouse-client --password" if you've set up a password.
```

**Comment by dariosusman** [comment]
apt-key shouldn't be used any more. The gpg/pgp key must be downloaded to  /etc/apt/trusted.gpg.d/, either in binary (.gpg) or armoured/ascii (.asc) format.

I ended up doing this:
```
# gpg --keyserver "hkps://keyserver.ubuntu.com:443" --recv-keys 8919F6BD2B48D754
# gpg -a --export 8919F6BD2B48D754  > /etc/apt/trusted.d/clickhouse.asc
# gpg --delete-key 3A9EA1193A97B548BE1457D48919F6BD2B48D754
# apt update
```
The last `gpg` line is to delete the key from root's keyring in this case.
Then apt will run smoothly. You just have to accept the new apt key 

Cheers,
Dario Susman

**Comment by DanRoscigno** [comment]
@Felixoid can you please have a look?

**Comment by Felixoid** [comment]
I'd say for now it's a low priority, since the method still works on every debian/ubuntu distro. But for sure, eventually, I'll implement the command to have the key downloaded properly.

**Comment by DanRoscigno** [comment]
Agreed, just letting you know.  I saw somewhere that some of the commands related to apt-key would be deprecated last October, but the ones we are using work fine.  Thanks.
